### DISCUSSION TITLE
Xiaomi Wireless button WXKG01LM vs YTC4040GL

### DISCUSSION THREAD

**Discussion by maurosmartins** [Koenkk/zigbee2mqtt#10752] [Devices]
Hello all, 

I've recently started using zigbee2mqtt, and I've successfully added a wireless button from Xiaomi WXKG01LM, as the one present on the supported device list. Nevertheless now that I want to buy more I don't find them readily available, instead I found YTC4040GL model.

does anyone know what the differences are and if this model is equally supported by zigbee2mqtt?

looking forward your reply, 
Best regards, Mauro.

**Comment by antonio1475** [comment]
Hi. For anyone reading this: it's the same thing. It's even listed in the whitelabel section of https://www.zigbee2mqtt.io/devices/WXKG01LM.html

I also bought one lsst week and it's identified in Z2M (and works):
![2A932AA5-D49A-4E10-90B3-A79CCE36FA89](https://user-images.githubusercontent.com/63130724/177021927-5483b1bd-5774-47fd-a47a-a3c42a09fe32.jpeg)
